auto_cpufreq/core.py

```diff
@@ -267,13 +267,20 @@ def turbo(value: bool = None):
     """
     p_state = Path("/sys/devices/system/cpu/intel_pstate/no_turbo")
     cpufreq = Path("/sys/devices/system/cpu/cpufreq/boost")
+    amd_pstate = Path("/sys/devices/system/cpu/amd_pstate/status")
 
     if p_state.exists():
         inverse = True
         f = p_state
     elif cpufreq.exists():
         f = cpufreq
         inverse = False
+    elif amd_pstate.exists():
+        value = amd_pstate.read_text().strip()
+        if value == "active":
+            print("CPU turbo is controlled by amd-pstate-epp driver")
+            return False
+        # Should not reach here.
```

Comment:
I prefer to raise error here, but not sure about this project convention :)

Reply:
Because following code will require `f`, but `f` is None at this moment.

Reply:
I get what you're saying. Its true but since it never reaches there, it shouldn't matter, but I suppose for future proofing, we can unindent the `return False`